lib/ramble/ramble/application.py

```diff
@@ -1060,8 +1060,13 @@ def _define_commands(
                         if ramble.config.get("config:shell") in ["bash", "sh"]:
                             redirect += " 2>&1"
 
+                    if cmd_conf.run_in_background:
+                        bg_cmd = " &"
```

Comment:
Do you know if this is the same method of running a process in the background for other shells? Should this be at least guarded with a check that we're on sh / bash / csh?

Reply:
Good question. It seems this ampersand actually has pretty good support across shells ((t)csh, fish both support it, even powershell starting from version 6 has it: https://screenshot.googleplex.com/33qLeDJjhwFvVQr. The only one that's missing seems to be bat, but not sure how much we care about that ;)